app/script/conversation/ConversationRepository.coffee

```diff
@@ -1327,17 +1329,54 @@ class z.conversation.ConversationRepository
   ###
   _send_encrypted_message: (conversation_id, generic_message, payload, precondition_option = false) =>
     @logger.info "Sending encrypted '#{generic_message.content}' message to conversation '#{conversation_id}'", payload
-    @conversation_service.post_encrypted_message conversation_id, payload, precondition_option
-    .then (response) =>
-      @_handle_client_mismatch conversation_id, response
-      return response
-    .catch (error) =>
-      throw error if not error.missing
+    return Promise.resolve()
+      .then =>
+        conversation_et = @find_conversation_by_id conversation_id
+        if conversation_et.verification_state() is z.conversation.ConversationVerificationState.DEGRADED
+          users_with_unverified_clients = conversation_et.get_users_with_unverified_clients()
+          user_ids_with_unverified_clients = users_with_unverified_clients.map (user_et) ->
+            return user_et.id
```

Comment:
I would rather make the third param optional and move this logic to `@_grant_outgoing_message`